modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/WorkflowAsset.macro

```diff
@@ -38,7 +38,8 @@ definition {
 			value1 = ${newPassword});
 
 		Type(
-			locator1 = "TextInput#ENTER_AGAIN",
+			key_text = "Reenter",
+			locator1 = "TextInput#ANY",
```

Comment:
Just using a generic path, since the label text was changed to "Reenter Password".